CRM/Contact/BAO/Group.php

```diff
@@ -448,7 +456,7 @@ public static function create(&$params) {
       }
```

Comment:
This block (see above) will actually force in a `parents` param, so we either need to alter `$parentsParamProvided` here or else modify the check below for refreshing the cache.

Reply:
Ok, any preference ? we could also go back to a safer approach where the cache is updated no matter what.